tests/gate_data.py

```diff
@@ -438,3 +438,37 @@ def DoubleExcitationMinus(phi):
     U[12, 12] = c
 
     return U
+
+
+def QuantumNumberPreservingOR(theta):
+    r"""Quantum number preserving four-qubit one-parameter gate.
+
+    Args:
+        theta (float): rotation angle
+    Returns:
+        array: the four-qubit matrix describing the operation
+    """
+
+    c = math.cos(theta / 2)
+    s = math.sin(theta / 2)
+
+    return np.array(
+        [
+            [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
```

Comment:
Some of the signs are different than those in the paper, please see my comment above.

Reply:
Done.